lib/internal/url.js

```diff
@@ -196,16 +188,12 @@ function onParsePathComplete(flags, protocol, username, password,
 
 function onParseSearchComplete(flags, protocol, username, password,
                                host, port, path, query, fragment) {
-  if (flags & binding.URL_FLAGS_FAILED)
-    return;
```

Comment:
Just to make sure I’m not missing something – these are dropped because the `on*Complete` functions are not called in the case of errors anymore, right?